src/routes/+page.svelte

```diff
@@ -1,18 +1,26 @@
 <script lang="ts">
 	import { page } from '$app/stores';
-
 	import { supabase } from '$lib/auth';
 	import Link from '$lib/components/Link.svelte';
 	import Button from '$lib/components/Button.svelte';
 </script>
 
 <div class="container">
-	<h1>Welcome to Annuaire</h1>
-
-	<Link href="inscription">S'enregistrer</Link>
-	<br />
-	<Link href="connexion">Se connecter</Link>
-	<br />
+	<nav class="navbar">
```

Comment:
You should create a components for the nav and put the related styles (in sass) in its file.
It's would be more maintainable, cleaner and easy to read.